app/models/like.rb

```diff
@@ -1,4 +1,4 @@
 class Like < ApplicationRecord
-  belongs_to :author, class_name: 'User'
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
   belongs_to :post, counter_cache: :likes_counter
```

Comment:
- [ ] Likes should also have a method that updates like counter

Reply:
@VICTOR-CHUKWU  it does have a counter and comments does also. Likes does not have likes that's why it's nill, but if you Comment.all you'll see